### PR TITLE
Update GPO designated receiver information

### PR DESCRIPTION
**Pull request by matthinz**
- Ask folks to sign up directly or ask #login-team-ada for questions
- Describe how to update the PII in config

### CODE REVIEW

**Comment by matthinz** [comment]
Preview is here: https://federalist-7d0b2b76-42fc-4df1-9825-8c3cd77e0a15.sites.pages.cloud.gov/preview/18f/identity-handbook/matthinz/11908-gpo-designated-receiver/articles/gpo-designated-receiver.html

**Review by jmax-gsa — CHANGES_REQUESTED** [review]
Can we emphasize the link to the spreadsheet a little? I missed it on first reading. Perhaps we could just repeat it in the 'What should I expect and how do I do it?' section.

**Review by solipet — APPROVED** [review]
LGTM!

**Comment by solipet** [comment]
> Can we emphasize the link to the spreadsheet a little? I missed it on first reading. Perhaps we could just repeat it in the 'What should I expect and how do I do it?' section.

I might argue to rename that section simply "What should I expect" since the stuff at the bottom explains how. The spreadsheet is linked both in the "How and why?" section and in Matt's new content.

File: _articles/gpo-designated-receiver.md [hunk 1]
```diff
@@ -32,4 +32,37 @@ It takes 3-7 business days for letters to arrive. It is common to not receive an
...
-If you'd like to volunteer to receive and record letters, don't hesitate to post in #login-appdev or add an agenda item to the [Engineering Huddle](https://docs.google.com/document/d/1g_V2vlT79tScBKIVw7M4UXf15TrG69iUrLHE0yE1GGI/). The information is stored in the secrets storage under `gpo_designated_receiver_pii` and requires the keys `first_name`, `last_name`, `address1`, `city` `state` and `zipcode`.
+If you'd like to volunteer to receive and record letters, please add your name to the [Upcoming Volunteers tab][upcoming-volunteers] of the GPO Designated Receiver Report spreadsheet. If you have any questions about the process, you can ask them in Slack in [#login-team-ada][team-ada-slack].
+
+## How to change the designated receiver
+
+Team Ada's oncall engineer updates the designated receiver configuration on (or around) the 15th of every month. Here's the process:
+
+### 1. Get the new designated receiver's address
+
+Remember: **names and addresses are PII**. When reaching out to the new receiver, please request that they supply the information to you via an ephemeral channel (typically Google Chat or over a video call).
```
Comment (zachmargolis) [inline _articles/gpo-designated-receiver.md:43]: I usually ask for a private Google Doc with the info that they should delete after

Comment (matthinz) [reply _articles/gpo-designated-receiver.md]: Good point! Updated in d3fc407

**Comment by matthinz** [comment]
> Can we emphasize the link to the spreadsheet a little? I missed it on first reading. Perhaps we could just repeat it in the 'What should I expect and how do I do it?' section.

As an inveterate skimmer, I agree with this and added a link to the "What should I expect..." section.

**Review by jmax-gsa — APPROVED** [review]
LGTM

**Pull request merged**